elide-contrib/elide-swagger/src/main/java/com/yahoo/elide/contrib/swagger/SwaggerBuilder.java

```diff
@@ -604,7 +607,17 @@ private boolean lineageContainsType(PathMetaData other) {
      * @param info Basic service information that cannot be generated
      */
     public SwaggerBuilder(EntityDictionary dictionary, Info info) {
+        this(dictionary, info, true);
+    }
+
+    /**
+     * @param dictionary The entity dictionary
+     * @param info Basic service information that cannot be generated
+     * @param supportLegacyDialect Whether or not to support the legacy dialect
+     */
+    public SwaggerBuilder(EntityDictionary dictionary, Info info, boolean supportLegacyDialect) {
```

Comment:
Since this is a builder, why is there no
```
    . supportLegacyDialect(boolean)
```

Reply:
So does this also need to be exposed as an configuration option for **elide-spring-boot-starter** or change the autoconfiguration to disable the legacy dialect?

Reply:
The auto-configuration has the legacy dialect disabled by default.  Also, you can auto-configure the Swagger object construction to turn on/off the dialects you want.